src/SDKs/Dns/Management.Dns/Microsoft.Azure.Management.Dns.csproj

```diff
@@ -6,12 +6,12 @@
 	<PropertyGroup>
 		<PackageId>Microsoft.Azure.Management.Dns</PackageId>
 		<Description>Provides management capabilities for Azure Dns.</Description>
-		<Version>2.0.0-preview</Version>
+		<Version>2.2.0-preview</Version>
```

Comment:
@muwaqar why move to 2.2 instead of 2.1?

Reply:
2.1 is taken by 2017-10-01 API version (changes for which should be in psSdkJson6 branch already).

Reply:
Don't know why GitHub is showing 2.0.0-preview (as before). The latest file (https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Dns/Management.Dns/Microsoft.Azure.Management.Dns.csproj) has 2.1.0-preview version.